src/lib.rs

```diff
@@ -285,7 +294,7 @@ fn find_config_files(
 /// RateLimit means we need to wait and retry, other things are errors we report
 #[derive(Debug)]
 enum ConfigFetchIssue {
-    NoConfigFound,
+    NoConfigFound(String),
```

Comment:
this isn't necessary, the repo url is known at the point where these messages are received.